test/parallel/test-stream-readableListening-state.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+/**
+ * Testing Readable Stream internal state
+ * `readableListening`
+ */
+
+const commun = require('../common.js');
+const assert = require('assert');
+const stream = require('stream');
+
+/**
+ * Fiest test escenary, were we are going to call  the
```

Comment:
I'm not sure what this is supposed to say. Also, there is an extra space before `the`.